api_examples/graphql/queries/list_smart_folders.graphql

```diff
@@ -1,9 +1,37 @@
+# Description
+# -----------
+#
+# The query return list of smart folders in turbot console.
+#
+
+# Usage
+# -----
+#
+# turbot graphql --query ./list_resources_by_resource_type_in_specific_account.graphql
+#
+
+# Filter
+# ------
+#
+# resourceType:
+#   Filter resources of a specific resource type.
+#   In example, the resources 'tmod:@turbot/turbot#/resource/types/smartFolder' returns list of smart folders existing in turbot console.
+#
+
+# Documentation
+# -------------
+#
+# For full documentation see:
+# - Filter documentation: https://turbot.com/v5/docs/reference/filter
+# - GraphQL resources: https://turbot.com/v5/docs/reference/graphql/query/resources
+#
+
 query ListSmartFolders {
-  resourceList(filter: "resourceType:'tmod:@turbot/turbot#/resource/types/smartFolder'") {
+  resources(filter: "resourceType:'tmod:@turbot/turbot#/resource/types/smartFolder'") {
```

Comment:
Should be resourceTypeId rather than `resourceType`

Can you fix all of those too?

Reply:
sure.